src/Symfony/Component/Notifier/Bridge/KazInfoTeh/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+5.4
```

Comment:
@symfony/mergers this PR must be against 6.1 branch, right?

Reply:
Yes